resolwe_bio/docker_images/utils/packages-python3.txt

```diff
@@ -4,6 +4,7 @@ pandas==0.23.0 --hash=sha256:29960ea1783c59fe86b323799b130592149e0f7bb739149e7fc
 requests==2.18.4 --hash=sha256:6a1b267aa90cac58ac3a765d067950e7dbbf75b1da07e895d1f594193a40a38b
 resdk==4.0.0 --hash=sha256:19b5409e9ac4e68fdd9fa8f15b295c1eb417842fa72fb605b143390451e7fedf
 scipy==1.1.0 --hash=sha256:729f8f8363d32cebcb946de278324ab43d28096f36593be6281ca1ee86ce6559
+multiqc==1.6.0 --hash=sha256:1e9f0480b7f6590a4ac159f69c1ee6c04aed0e3c156538b4994e3df451a6244e
```

Comment:
You could put this entry a few rows higher, to keep alphabetical ordering. Not urgent.